### PR TITLE
[ET-835] Fix CSS in Birth Date Field not applying in all contexts

### PR DESCRIPTION
**Pull request by Luc45**
Ticket: https://moderntribe.atlassian.net/browse/ET-835

### CODE REVIEW

File: src/resources/postcss/tickets.pcss [hunk 1]
```diff
@@ -27,6 +27,60 @@
...
+		display: inline-block;
+
+		@media(--viewport-medium) {
+			width: calc( ( 100% / 3 ) - var(--spacer-1) - 4px ); /* 4px = safety margin */
+			margin-right: var(--spacer-1);
+			display: inline-block;
+		}
+
+		select {
+			border-color: var(--form-color-border-secondary);
```
Comment (paulmskim) [inline src/resources/postcss/tickets.pcss:50]: You can set border-color, border-style, and border-width all in one border style.

File: src/resources/postcss/tickets.pcss [hunk 2]
```diff
@@ -27,6 +27,60 @@
...
+			}
+			@mixin mobile-body-2;
+			appearance: none;
+			background-color: var(--color-background);
+			background-image: url('../images/select.png');
+			background-position: right center;
+			background-repeat: no-repeat;
+			background-size: 33px 33px;
+			height: auto;
+			padding: 0.7em;
```
Comment (paulmskim) [inline src/resources/postcss/tickets.pcss:75]: I would not mix px and em, but rather stick to px

File: readme.txt [hunk 3]
```diff
@@ -117,9 +117,10 @@ Currently, the following add-ons are available for Event Tickets:
 
 == Changelog ==
 
-= [TBD] =
+= [TBD]  =
```
Comment (sc0ttkclark) [inline readme.txt:120]:
```suggestion
= [TBD] TBD =
```

File: src/resources/postcss/tickets.pcss (hunk 1 above)
Comment (cliffordp) [reply src/resources/postcss/tickets.pcss:50]: fixed

File: src/resources/postcss/tickets.pcss (hunk 2 above)
Comment (cliffordp) [reply src/resources/postcss/tickets.pcss]: done

**Comment by cliffordp** [comment]
How it looks now: https://share.getcloudapp.com/DOuAGWlY

**Comment by cliffordp** [comment]
How it looks after my additional changes:

* Over 960 (full) -- same 50% width rule as text input fields
image
* Tablet 
image
* Phone 
image

File: src/views/registration-js/attendees/fields/birth.php [hunk 4]
```diff
@@ -33,47 +33,52 @@
 		<label
 			class="tribe-common-b2--min-medium tribe-tickets-meta-label"
 			for="<?php echo esc_attr( $option_id ); ?>"
-		><?php echo wp_kses_post( $field['label'] ); ?><?php tribe_required_label( $required ); ?></label>
+		>
+			<?php echo wp_kses_post( $field['label'] ); tribe_required_label( $required ); ?>
```
Comment (tr1b0t) [inline src/views/registration-js/attendees/fields/birth.php:37]: * :no_entry_sign: Inline PHP statement must contain a single statement; 2 found
* :no_entry_sign: Each PHP statement must be on a line by itself

<sub>hash: c13f635fb0cf1991f1b2ad7a4408162b</sub>

Comment (cliffordp) [reply src/views/registration-js/attendees/fields/birth.php]: fixed

File: src/views/registration-js/attendees/fields/birth.php [hunk 5]
```diff
@@ -33,47 +33,52 @@
 		<label
 			class="tribe-common-b2--min-medium tribe-tickets-meta-label"
 			for="<?php echo esc_attr( $option_id ); ?>"
-		><?php echo wp_kses_post( $field['label'] ); ?><?php tribe_required_label( $required ); ?></label>
+		>
+			<?php echo wp_kses_post( $field['label'] ) . ' ' . tribe_required_label( $required ); ?>
```
Comment (tr1b0t) [inline src/views/registration-js/attendees/fields/birth.php:37]: * :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'tribe_required_label'.
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: 61d78444e4aa51de65feb327f9893cab</sub>

**Comment by cliffordp** [comment]
@sc0ttkclark failed tests don't seem relevant to this ticket. must be from base branch. All are "Restv1 Tests" type.

File: readme.txt [hunk 6]
```diff
@@ -119,9 +119,10 @@ Currently, the following add-ons are available for Event Tickets:
 
 = [TBD] TBD =
 
-* Feature - For the `[tribe-user-event-confirmations]` shortcode, link to both the event and the direct My Tickets page. [ET-792]
 * Fix - Resolve a PHP notice when rendering the Event Tickets Plus field for Birth date [ETP-330]
+* Tweak - Improve the Birth date field styles that could show up without styling under certain contexts. [ET-835]
 * Tweak - Add hook to ticket email template to allow injecting content between ticket details and organizers. [ET-844]
+* Feature - For the `[tribe-user-event-confirmations]` shortcode, link to both the event and the direct My Tickets page. [ET-792]
```
Comment (sc0ttkclark) [inline readme.txt:125]:
```suggestion
* Tweak - For the `[tribe-user-event-confirmations]` shortcode, link to both the event and the direct My Tickets page. [ET-792]
```

**Pull request merged**